### PR TITLE
[ADD] module web_dialog_size

### PR DESCRIPTION
**Pull request by anthony-muschang**
This PR add a new module that let the user expand a dialog box to the full screen width and restore it to the default size.

It is named web_dialog_size as it could be extended to propose other dialog size management feature.

Dialog box at the normal size:

dialog-size-normal

Dialog box extended:

dialog-size-expand

### CODE REVIEW

File: web_dialog_size/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+#     GNU Affero General Public License
+#     along with web_expand_dialog.
+#     If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': "Web Dialog Size",
+
+    'summary': """
+        A module that let the user expand a
```
Comment (hbrunn) [inline web_dialog_size/__openerp__.py]: letS

**Comment by hbrunn** [comment]
type in summary, all else :+1:

**Comment by anthony-muschang** [comment]
Thanks for, the review. The typo is fixed.

**Comment by anthony-muschang** [comment]
Note: The build pep8 failed due to the following pep8 error:
./web_graph_improved/**init**.py:1:1: W391 blank line at end of file.

Which come from the merge of the following PR: https://github.com/OCA/web/pull/74

**Comment by sbidoul** [comment]
:+1: (manual test and code review)

Thanks @anthony-muschang !

**Comment by sbidoul** [comment]
cc/ @chrisdec

**Comment by legalsylvain** [comment]
flake8 : ./web_graph_improved/**init**.py:1:1: W391 blank line at end of file
Otherwise, :+1: (Code review. No test.)

**Comment by anthony-muschang** [comment]
Thanks for the review
@legalsylvain This issue is fixed by the PR #87

**Comment by adrienpeiffer** [comment]
:+1: (code review)

**Pull request merged**